app/src/main/java/io/appium/uiautomator2/model/api/gestures/DragModel.java

```diff
@@ -0,0 +1,35 @@
+package io.appium.uiautomator2.model.api.gestures;
+
+import android.graphics.Point;
+
+import io.appium.uiautomator2.model.RequiredField;
+import io.appium.uiautomator2.model.api.BaseModel;
+import io.appium.uiautomator2.model.api.ElementModel;
+
+public class DragModel extends BaseModel {
+    public ElementModel origin;
+    public Double startX;
+    public Double startY;
+    @RequiredField
+    public Double endX;
+    @RequiredField
+    public Double endY;
+    public Integer speed;
+
+    public DragModel() {}
```

Comment:
Codacy found an issue: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/dpgraham/appium-uiautomator2-server/pullRequest?prid=6222243)